src/goto-instrument/mmio.h

```diff
@@ -14,6 +14,8 @@ Date: September 2011
 #ifndef CPROVER_GOTO_INSTRUMENT_MMIO_H
 #define CPROVER_GOTO_INSTRUMENT_MMIO_H
 
+#include <analyses/guard.h>
```

Comment:
That's surprising?!

Reply:
This has been left from old commits, which should have been cleaned-up after rebase. I'll remove them.